controllers/storagecluster/external_resources.go

```diff
@@ -261,6 +261,10 @@ func (obj *ocsExternalResources) ensureCreated(r *StorageClusterReconciler, inst
 			}
 		}
 
+		if instance.Status.Phase == statusutil.PhaseReady {
```

Comment:
We should not check for the status.phase at all, We should just call it and return only if there is an error.

This will create a problem where we will not go ahead with the below code if the status is already ready.

Reply:
Updated in latest commit